### PR TITLE
feat(sidemenue admindashboard page)

### LINKED ISSUE #15: navigation menu

### PR DESCRIPTION
**Pull request by ikoworld**
add the logout function and fetching admin profile information from firestore database 

#15

# Related Issue

- Resolve #15

### CODE REVIEW

File: src/components/Sidemenu.jsx [hunk 1]
```diff
@@ -14,16 +17,53 @@ import Dashboard from "./Dashboard";
...
     const [mounted, setMounted] = useState(false);
     const currentTheme = theme === "system" ? "light" : theme; // State for the current theme
+    const [userData, setUserData] = useState(null);
+
     useEffect(() => {
-        setMounted(true);
+        const fetchInformation = async () => {
+            if (auth.currentUser) {
+                const userId = auth.currentUser.uid;
+                console.log("userId:", userId);
```
Comment (merzoukfatima) [inline src/components/Sidemenu.jsx]: remove console log

File: src/components/Sidemenu.jsx [hunk 2]
```diff
@@ -14,16 +17,53 @@ import Dashboard from "./Dashboard";
...
+                const docRef = doc(db, "users", userId);
+
+                try {
+                    const docSnap = await getDoc(docRef);
+                    if (docSnap.exists()) {
+                        const userData = { ...docSnap.data(), id: userId };
+                        setUserData(userData);
+                    } else {
+                        console.log("Document does not exist.");
```
Comment (merzoukfatima) [inline src/components/Sidemenu.jsx]: remove console log

File: src/components/Sidemenu.jsx [hunk 3]
```diff
@@ -14,16 +17,53 @@ import Dashboard from "./Dashboard";
...
+
+        fetchInformation();
     }, []);
+
     if (!mounted) return null;
 
+    const handleLogout = async () => {
+        try {
+            await signOut(auth);
+            console.log("you are logged out");
```
Comment (merzoukfatima) [inline src/components/Sidemenu.jsx]: remove console log

File: src/images/placeholderImage.png
Comment (merzoukfatima) [inline src/images/placeholderImage.png:1]: add sidemenu.test.jsx please @ikoworld

**Review by merzoukfatima — CHANGES_REQUESTED** [review]

**Review by walidbelharazem — APPROVED** [review]
Consider adding comments to facilitate the code review ; Otherwise, the code functions smoothly.

**Review by merzoukfatima — APPROVED** [review]

File: src/images/placeholderImage.png
Comment (ikoworld) [reply src/images/placeholderImage.png:1]: i will create new issue for the test and translate

File: src/components/Sidemenu.jsx (hunk 3 above)
Comment (ikoworld) [reply src/components/Sidemenu.jsx]: done

**Pull request merged**